I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 108, in script call
    call check from _call_check
  File "game/script.rpy", line 82, in script call
    call game3 from _call_game3
  File "game/script.rpy", line 2484, in script
    show seng_supred with Dissolve(6.0f)
  File "game/script.rpy", line 2484, in <module>
    show seng_supred with Dissolve(6.0f)
NameError: name 'rgiht' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 108, in script call
    call check from _call_check
  File "game/script.rpy", line 82, in script call
    call game3 from _call_game3
  File "game/script.rpy", line 2484, in script
    show seng_supred with Dissolve(6.0f)
  File "C:\renpy-7.2.2-sdk\renpy\ast.py", line 1169, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\renpy-7.2.2-sdk\renpy\ast.py", line 1128, in show_imspec
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\renpy-7.2.2-sdk\renpy\python.py", line 1961, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\renpy-7.2.2-sdk\renpy\python.py", line 1954, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 2484, in <module>
    show seng_supred with Dissolve(6.0f)
NameError: name 'rgiht' is not defined

Windows-8-6.2.9200
Ren'Py 7.2.2.491
버드나무 꽃 만개할 그날까지 1.1
Thu Jun 13 11:07:26 2019
